apps/oxlint/src-js/plugins/scope.ts

```diff
@@ -254,6 +259,7 @@ export function getScope(node: ESTree.Node): Scope {
       return scope.type === 'function-expression-name' ? scope.childScopes[0] : scope;
     }
 
+    // @ts-expect-error - Don't want to create a new variable just to make it nullable
```

Comment:
I made an issue for a minifier enhancement for this case:

https://github.com/oxc-project/oxc/issues/15908